src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -0,0 +1,137 @@
+(ns status-im.ui.screens.profile.user.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [status-im.ui.screens.profile.user.styles :as styles]
+            [status-im.ui.components.toolbar.view :as toolbar]
+            [status-im.ui.components.react :as react]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.common.styles :as common.styles]
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.i18n :as i18n]
```

Comment:
We should agree on alphabetical ordering :)

Reply:
I'm all for consistency, but does it really matter for requires?

Reply:
Sometimes you can spot duplications or incorrect naming this way. Certainly helps my OCD!
Maybe just something to keep in mind for later?

Reply:
I agree, I just think it's a waste of time to do that manually. We can try and figure out how to set the tools